syntax = "proto3";

package latency;

service LatencyTest {
  rpc MeasureTime(TimeRequest) returns (TimeResponse) {}
}

message TimeRequest {
  int64 client_send_time = 1;
  bytes payload = 2;
}

message TimeResponse {
  int64 server_receive_time = 1;
  bytes payload = 2;
}